src/core/caniuse.js

```diff
@@ -200,8 +200,20 @@ function createTableHTML(conf, stats) {
       .reverse();
     const { support, title } = getSupport(latestVersion);
     const cssClass = `caniuse-cell ${support}`;
+    let emoji;
```

Comment:
As I mentioned in the original bug, it should be possible to do this without modifying the caniuse.js file by using CSS Generated content: an example where an emoji is added https://www.viget.com/articles/css-generated-content/